src/app/shared/org-logo.service.ts

```diff
@@ -0,0 +1,7 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class OrgLogoService {
+  // default org logo image URL, including enough parent directory references to ensure that '/assets' is always relative to the root
+  readonly DEFAULT_URL = '../../../../assets/images/dockstore/default-org-logo.svg';
```

Comment:
This may be made moot in prod by the string substitution we do for the CDN (?), but this string is used by different components with different urls that may not be at the same depth level. So `../../../../assets` may take you to the assets directory in some cases but not in others? Just an untested theory, but the relative path should be resolved relative to the current url in the browser.

Reply:
The initial goal here was include enough `../`s to crawl up the url path so that, during development, we'd reach root in all cases (four levels), and to provide the deploy script with the pattern it latches onto when it does the rewrite (which, as you mention, makes the issue moot):

https://github.com/dockstore/dockstore-ui2/blob/d10573defb7d9dfc106cb19e418145746a43eebe/.circleci/config.yml#L218

However, upon further inspection, we appear to always render with `<base href="/">` in the header, which means that all relative urls are resolved relative to the root url, and the `../`s don't matter (except to the deploy script).  Indeed, if you add or subtract any number of `../`s from the images on your local setup, you will find they still work (which is good, because a lot of them don't include the correct number of `../`s in the first place).

In summary, might as well leave it as is.